src/utils/storage.js

```diff
@@ -0,0 +1,8 @@
+export const COUNTER = "COUNT";
+
+export const setLocalData = (key, value) =>
+	localStorage.setItem(key, JSON.stringify(value));
+
+export const getLocalData = key => {
+	return JSON.parse(localStorage.getItem(key)) || {};
```

Comment:
the else condition should be inside the parse func else when the parse gets an empty/null data in the getitem it will throw an error

Reply:
return JSON.parse(localStorage.getItem(key) || {}); -> This is giving error

Reply:
"{}" instead